recipes/newrelic/nginx/suse.yml

```diff
@@ -0,0 +1,123 @@
+name: NGINX Open Source Integration
+description: New Relic install recipe for default NGINX Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-nginx
+
+installTargets:
+  - type: host
+    os: linux
+    platformFamily: "suse"
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - nginx
+
+# Examine Infrastructure events for correlated data
+processMatch:
+  - /nginx/
+
+# Examine Metrics, Events, and Logging for correlated data
+# Used by the UI to determine if you've successfully configured and are ingesting data
+meltMatch:
+  metrics:
+    pattern:
+      - /net.connectionsActive/
+      - /net.connectionsAcceptedPerSecond/
+      - /net.connectionsDroppedPerSecond/
+      - /net.connectionsReading/
+      - /new.connectionsWaiting/
+      - /net.connectionsWriting/
+      - /net.requestsPerSecond/
+      - /software.edition/
+      - /software.version/
+  logging:
+    pattern:
+      - /http/
```

Comment:
I think just `http` is sufficient here.  The CLI doesn't yet implement this, but I'm expecting that we pass the string directly to infra agent config.

Reply:
Hey @zlesnr , thanks for the feedback.
Yeah I actually have no idea about all those meltMatch data. I've copied it from another definition I believe. I'll cut most of it out.